proposed/container-configuration-meta.md

```diff
@@ -29,10 +29,62 @@ The way a framework can automatically **discover the container definitions** of
 
 ## 4. Approaches
 
-### 4.1. The format
+### 4.1. Features
 
-TODO
+First of all, a word of caution. Container configuration is shared between the application developer and the packages bringing some configuration. Ultimately, container configuration is the responsibility of the application developer. Packages should however be able to bring a "default" configuration to help the developer getting started with sensible defaults (container configuration can be quite complex sometimes). This configuration should be both optional (possibility to bypass it completely), and overwritable (possibility to changes bits of the configuration).
+
+In this section, we will describe the list of features that we can consider to fulfill the goal of this PSR: packages registering container entries. Let's list all possible features and later, we can vote on the features we want to keep or not.
+
+1. Ability to create a container entry using the `new` keyword.
+
+1. Ability to call methods (setters or otherwise) on a container entry.
+
+1. Ability to set public properties of a container entry.
```

Comment:
I think you should add another list item "Ability to set private and protected properties of a container entry."

I'm not saying we should do this, I think I've seen some containers that do.